src/demos/AirshipModal.tsx

```diff
@@ -103,15 +105,16 @@ export class AirshipModal extends React.Component<Props, State> {
           function upValue(keyboardHeight: number): number {
             return Math.max(keyboardHeight, downValue)
           }
-
+          const styleOverride = this.props.styleOverride || {}
           return (
             <KeyboardTracker downValue={downValue} upValue={upValue}>
               {(keyboardAnimation, keyboardLayout) =>
                 this.renderModal(
                   this.state.height,
                   metrics,
                   keyboardAnimation,
-                  keyboardLayout
+                  keyboardLayout,
+                  styleOverride
```

Comment:
Actually, since we are just passing it through, `renderModal` itself can grab the styles, and we can leave `render` un-touched.